.github/workflows/ci.yml

```diff
@@ -146,3 +146,24 @@ jobs:
     - name: Create source and wheel dist
       run: |
         python -m build
+
+  server-package:
+    # This job checks if we can build our server package
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./server
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build the Docker image
+        run: |
+          docker buildx build -t basyx-python-sdk-http-server .
```

Comment:
Btw, is there a reason the image is called `basyx-python-sdk-http-server` and not just `basyx-python-server`?

Reply:
Not really, I just forgot to change the name.